packages/components/src/Components/InsightsLabel/InsightsLabel.test.js

```diff
@@ -0,0 +1,19 @@
+import InsightsLabel from './InsightsLabel';
+import React from 'react';
+import { shallow } from 'enzyme';
+import toJson from 'enzyme-to-json';
+
+describe('InsightsLabel component', () => {
+    it('should render with icon', () => {
+        const wrapper = shallow(<InsightsLabel value={1} showIcon/>);
```

Comment:
I thinks these tests are out of the sync: you have `showIcon` here. The test below does the same.

I would switch to explicitly checking if the icon is rendered or not: `expect(wrapper.find(Icon)).toHaveLenght(0)` or you can check if the PF label gets the right props.